source/unified-test-format/unified-test-format.rst

```diff
@@ -2236,8 +2236,8 @@ Clear the entity map for this test. For each ClientSession in the entity map,
 the test runner MUST end the session (e.g. call
 `endSession <../sessions/driver-sessions.rst#endsession>`_).
 
-If the test failed, the test runner MUST terminate any open transactions (see:
-`Terminating Open Transactions`_).
+If the test failed or started a transaction, the test runner MUST terminate
+any open transactions (see: `Terminating Open Transactions`_).
```

Comment:
If you're always going to kill all sessions after any test that starts a transaction, including failed tests here seems redundant. A failed test that never starts a transaction would never leave behind any transactions on its own.